ocs_ci/framework/main.py

```diff
@@ -166,6 +171,10 @@ def process_ocsci_conf(arguments):
         utils.add_path_to_env_path(framework.config.RUN["bin_dir"])
     if args.disable_components:
         framework.config.ENV_DATA["disable_components"] = args.disable_components
+    if args.default_cluster_context_index:
+        framework.config.ENV_DATA[
+            "default_cluster_context_index"
+        ] = args.default_cluter_context_index
```

Comment:
I see you are storing this in ENV_DATA in config - but in which config it will be stored? In common on all contexts? Or it will be only in conf context 0?

Reply:
According to the comment in `ocs_ci/framework/__init__.py` it will be stored in all of the configs since it is a common argument.

Reply:
> According to the comment in `ocs_ci/framework/__init__.py` it will be stored in all of the configs since it is a common argument.
> 
> Edit: @shylesh Doesn't that mean we need to iterate over each config and set that value explicitly here?

Yes, common args will be available in all contexts because we would have merged common args with every cluster args during parsing.